tests/js/web/FocusManagementServiceTest.js

```diff
@@ -0,0 +1,119 @@
+import $ from 'jquery';
+import FocusManagementService from '../../../js/FocusManagementService'
+
+describe('FocusManagementService', () => {
+    let $html;
+    let $body;
+    let $link;
+    let $input;
+    let $select;
+    let $textarea;
+    let $button;
+    let focusManagementService;
+
+    beforeEach(() => {
+        $html = $('html');
+        $body = $('body');
+        $link = $('<a href="#">Link</a>').appendTo($body);
+        $input = $('<input type="text"/>');
+        $select = $('<select><option value="one">One</option></select>');
+        $textarea = $('<textarea>text</textarea>');
+        $button = $('<button>Button</button>');
+
+        focusManagementService = new FocusManagementService();
+    });
+
+    afterEach(() => {
+        $body.empty();
+    });
+
+    describe('storeElement()', () => {
+        it('should store the element', () => {
+            focusManagementService.storeElement($link);
+
+            expect(focusManagementService.$element === $link).to.be.true;
```

Comment:
```expect(focusManagementService.$element).to.be($link)``` might make the error output easier to reason about if this should fail in the future